### PR TITLE
Add some guidelines to repo and add code to delete .ini files (AKA games from flashcard)

### PR DESCRIPTION
**Pull request by Jolty95**
#### What's new?

I've added a new submodule and a new script to push updates and some github files.
Ussage:
you can modifiy TWLoader-update folder that is inside TWLoader repo (remember that compiling.bat will copy build heree too)
after modifying, use updateTWLoader-Update.bat to copy TWLoader-update folder (updated) to ../TWloader-update and just commit it. (you can add `cd ../TWLoader-update` and `git push origin master` to auto-update if you want)
Also, I've added code to delete the .ini files from flashcard. To use it: press "Set as donor rom" in a game from flashcard location on TWLoader. .ini file will be deleted (it takes at least one second due to the huge amount of code that TWLoader have). There is a small bug that I haven't got time to fix. After deleting, they will be a n + 1 boxart in the right. I mean, if you have 5 roms, delete one, you have 4 roms but 5 boxarts. Reseting TWLoader fixes this (not a solution tbh)
Also, I haven't managed to change the "Set as donor rom" to "Delete .ini" because button struct is a `const` and buttons[3].title is a `const wchart*` too. Don't know how to archive this to change that if flashcard location is set and also added a clean.bat in TWLoader's GUI just because is easier (at least for me) en compileCIA.bat delete the "make clean" to don't clean at every nighty (I only `make clean` before sending PR's)
Remember: games for flashcard are NOT deleted. Only their .ini files. Also shall I include an option to delete games from SD? (delete the .nds files and banner and boxarts if they exist)
TODO: clean banner and boxarts for flashcard too. Didn't remember b4

#### What is fixed?

Re-align to left some small strings and remove an unused string in makefile and remove libraries from readme.md that are unussed.
Also fixed travis with latest citro3ds

#### Where have tested it?

New 3DS XL with A9LH+Luma+Sysnand on 11.2 
Old 3DS XL with A9LH+Luma+Sysnand on 11.4
(Latest commits of Luma)

*** 
#### Pull Request status
- [x]  This PR has been tested using latest DEVKITPRO, DEVKITARM, SFILLIB, SF2DLIB, LIBNDS and Citro3DS.  

_(Do not edit after this point)_
PS: This is the first so...I'll autocomplete it xD
- [x]  This PR is fully documented.
- [x]  This PR has been tested before sending.
- [x]  This PR follow C style and convention.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+*** 
+#### Pull Request status
+- [ ]  This PR has been tested using latest DEVKITPRO, DEVKITARM, SFILLIB, SF2DLIB, LIBNDS and Citro3DS.  
+
+_(Do not edit after this point)_
+- [ ]  This PR is fully documented.
+- [ ]  This PR has been tested before sending.
+- [ ]  This PR follow C style and convention.
```
Comment (Margen67) [inline .github/PULL_REQUEST_TEMPLATE.md]: follows

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+
+#### What is fixed?
+
+
+
+
+
+#### Where do you have tested it?
```
Comment (Margen67) [inline .github/PULL_REQUEST_TEMPLATE.md]: Change to `Where have you tested it?`

File: .github/contributing.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Contributing Guide
+
+Contributions are always welcome to TWLoader! If you have something new to share us, please, do it.
```
Comment (Margen67) [inline .github/contributing.md]: *share with

File: .github/contributing.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## Issues Guidelines
+
+Before you submit an issue, check that it meets these guidelines:
+
+- specify the version of TWLoader and TWLoader NAND Side you are using
+- specify what console are you using and your entrypoint
+- specify the CFW version you're using
+- specify if you're using an emunand
+- if the issue it's about a bug, please provide a minimal failing example and environment
```
Comment (Margen67) [inline .github/contributing.md]: change it's to is

File: .github/issue_template.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+<!---
+##### THIS IS A ISSUE TRACKER FOR TWLOADER. For support please go to:
```
Comment (Margen67) [inline .github/issue_template.md]: Change `a` to `the`

File: .github/contributing.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+3. Please test all your code and explain what you did before creating a PR. 
+4. When a PR pass this checks, we will tag it ready to merge and merge it as soon as possible.
+
+## Setting up your environment
+
+If you want to collaborate with us, you will need to install DEVKITPRO and DEVKITARM. You will also need xerpi's sfillib,sf2dlib, ahezard libnds and citro3ds. You also need makerom,make_cia and bannertool on system location.
+
+## Style
+
+All code must be clear and enough readable. Indentation is a must and code must follow C variable and name convention.
```
Comment (Margen67) [inline .github/contributing.md]: Remove `enough`

File: .github/issue_template.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+###### TWLoader GBATemp thread: https://gbatemp.net/threads/twloader-ctr-mode-nds-app.448375/
+###### Also check the Wiki (https://github.com/Robz8/TWLoader/wiki) before making an issue.
+###### Keep in mind that TWLoader is only a frontend for nds-bootstrap. DO NOT OPEN AN ISSUE FOR NOT LOADING GAMES THAT ARE NOT SUPPORTED BY NDS-BOOTSTRAP YET!
+###### Issues that doesn't attach any log file or any reproducible method will be closed.
```
Comment (Margen67) [inline .github/issue_template.md]: Change `doesn't` to `don't`

**Comment by Jolty95** [comment]
All fixed.
Thanks @Margen67 👍

File: .github/issue_template.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+### Steps to reproduce
+
+1.
+2.
+3.
+
+### Stack trace and console log
+
+If you are getting any crash of want to upload your log file, please,  write that information between the following quotation marks.
```
Comment (Margen67) [inline .github/issue_template.md]: Change `any` to `a`

Remove `of want to` and the double space around `please,  write`

Comment (Jolty95) [inline .github/issue_template.md]: That "of want to" should be a "or want to". It's ok like this?

File: .github/contributing.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+Before you submit a pull request from your forked repo, check that it meets these guidelines:
+
+1. If the pull request fixes a bug, tell us what test did you have checked before pushing.
+2. If the pull request adds functionality, you must tell us what's new and how to use it.
+3. Please test all your code and explain what you did before creating a PR. 
+4. When a PR pass this checks, we will tag it ready to merge and merge it as soon as possible.
+
+## Setting up your environment
+
+If you want to collaborate with us, you will need to install DEVKITPRO and DEVKITARM. You will also need xerpi's sfillib,sf2dlib, ahezard libnds and citro3ds. You also need makerom,make_cia and bannertool on system location.
```
Comment (Margen67) [inline .github/contributing.md]: *ahezard's

By `on system location` I think you meant on PATH, or installed.

Comment (Jolty95) [inline .github/contributing.md]: I've copied it from readme.mb here but in readme.md is also 's missing. I'll add it too
And yes, I mean in devkitpro environment. I'll change this too in readme and contributing.md

File: .github/contributing.md [hunk 10]
```diff
@@ -24,7 +24,7 @@ Before you submit a pull request from your forked repo, check that it meets thes
 
 ## Setting up your environment
 
-If you want to collaborate with us, you will need to install DEVKITPRO and DEVKITARM. You will also need xerpi's sfillib,sf2dlib, ahezard libnds and citro3ds. You also need makerom,make_cia and bannertool on system location.
+If you want to collaborate with us, you will need to install DEVKITPRO and DEVKITARM. You will also need xerpi's sfillib, sf2dlib, ahezard's libnds and citro3ds. You also need makerom on your devkitpro environment.
```
Comment (maorninja) [inline .github/contributing.md]: You make it seem like if ahezard owns both libnds and citro3ds. You should put citro3ds before libnds

Comment (Jolty95) [inline .github/contributing.md]: Yes, that's right too.
Sorry for that. In spanish at least only the first have that property (don't know how to explaint better)
I'll correct there and readme.md too

**Comment by Jolty95** [comment]
@Robz8 this can be merge if you want (ofc) :P 
Also, I'd like to know your opinion about .github files I've added in you can share it.

**Comment by Robz8** [comment]
You should let the X button delete games on SD card/flashcard.

**Comment by Robz8** [comment]
I'll merge this for now.

**Comment by Jolty95** [comment]
Robz I really don't wanna bother...
Between tomorrow/Sunday/Monday, I'll add:
-Delete games from SD (.nds/banner/boxart)
-Delete .ini files (banner and boxart)
-Confirmation dialog (?)

I think I don't need to reset cursor position in every deletion, so that can be a "plus" to final user.
And also, using a Vector I can add a "L+X" to select more than one games to be deleted (only in the same page for now)

**Pull request merged**